src/ml_board/frontend/Dashboard/src/components/graphs/Graph.tsx

```diff
@@ -0,0 +1,76 @@
+import { Box, Card, Grid } from "@mui/material";
+import { ChartOptions } from "chart.js";
+import { Line } from "react-chartjs-2";
+import { useAppSelector } from "../../app/hooks";
+import { selectChartLabelsById, selectExperimentsPerChartById } from "../../redux/charts/chartsSlice";
+import { selectColorMap } from "../../redux/status/statusSlice";
+// styles
+import styles from "./Graphs.module.css";
+
+// https://www.chartjs.org/docs/latest/general/data-structures.html
+
+export default function Graph({ chart_id }: { chart_id: string }) {
+
+    const chartLabels = useAppSelector(state => selectChartLabelsById(state, chart_id));
+    const experimentsDic = useAppSelector(state => selectExperimentsPerChartById(state, chart_id));
+    const colors = useAppSelector(selectColorMap);
+
+    // prepare data (Warning Looping!)
+    const data = {
+        labels: chartLabels, // the X-axis:Array<number>
+        datasets: !experimentsDic ? [] :
+            // loop over the experiments in the ChartF
+            Object.values(experimentsDic).map(exp => ({
+                label: "experiment_" + exp!.exp_id, // exp_name
+                data: exp!.data, // exp_values:Array<number>, Y-axis, same size as X-axis
+                fill: false,
+                backgroundColor: colors[exp!.exp_id],
+                borderColor: colors[exp!.exp_id],
+                tension: 0, // to give smoothness to the line curves
+            })),
+    };
+
+    // NOTE: casting as ChartOptions happens later, otherwise the radius parameters will give errors!!!
+    const options = {
+        plugins: {
+            title: {
+                text: chart_id,
+                display: true,
+                color: 'black',
+                font: {
+                    weight: 'bold',
+                    size: 20 //'20px'
+                }
+            },
+            legend: {
+                display: true,
+                labels: {
+                    usePointStyle: true,
+                    pointStyle: 'circle'
+                }
+            }
+        },
+        animation: {
+            duration: 300,
+            easing: 'linear'
+        },
+        radius: 3,  // radius of the label tag
```

Comment:
Is this the radius for a data point when it's rendered in the graph? Could we set it to 0 and see how it would look like? As long as we see a point while hovering I think it is fine and would also save some space on the DOM :)

Reply:
3 is actually the default value (at least on my screen😅) but 0 wasn't good at all (there were no indecations) so I ended up setting it to 1